rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -306,20 +363,40 @@ FindNodePeerMessage sendFindNode(Node node) {
         InetSocketAddress nodeAddress = node.getAddress();
         String id = UUID.randomUUID().toString();
         FindNodePeerMessage findNodePeerMessage = FindNodePeerMessage.create(this.key.getNodeId(), id, this.key, this.networkId);
+
+        logger.debug("sendFindNode - Sending find node message to " +
+                        "nodeAddress hostName: [{}], " +
+                        "nodeAddress hostName: [{}], " +
```

Comment:
```suggestion
                        "nodeAddress: [{}/{}], " +
```

Reply:
Done, I also updated all instances where we log the addres to follow this format.